examples/app-with-extension-point/docker-compose.yaml

```diff
@@ -7,7 +7,7 @@ services:
       context: ./.config
       args:
         grafana_image: ${GRAFANA_IMAGE:-grafana-enterprise}
-        grafana_version: ${GRAFANA_VERSION:-10.3.3}
+        grafana_version: ${GRAFANA_VERSION:-main}
```

Comment:
We should probably set this to a specific version instead of `main` since `main` is a moving target.